authsignal/client.py

```diff
@@ -0,0 +1,256 @@
+import decimal
+import authsignal
+import authsignal.version
+
+import json
+import requests
+_UNICODE_STRING = str
```

Comment:
Unless you were thinking of supporting Python2 I'm not sure what this would be used for?